LICENSES/README.md

```diff
@@ -0,0 +1,85 @@
+# systemd Project Licensing
+
+## Main License
+
+The systemd project uses a single-line reference to Unique License Identifiers
+in source files as defined by the Linux Foundation's SPDX project (https://spdx.org/).
+
+The current set of valid, predefined SPDX identifiers can be found on the SPDX
+License List at https://spdx.org/licenses/.
+
+The 'LICENSES/' directory contains all the licenses used by the sources included in
+the systemd project source tree.
+
+Unless otherwise noted, the systemd project sources are licensed under the terms
+and conditions of the **GNU Lesser General Public License v2.1 or later**.
+
+## Other Licenses
+
+The following exceptions apply:
+
+ * the following udev sources are licensed under **GPL-2.0-or-later**:
+   - src/udev/ata_id/*
+   - src/udev/cdrom_id/*
+   - src/udev/dmi_memory_id/*
+   - src/udev/mtd_probe/*
+   - src/udev/scsi_id/*
+   - src/udev/udevadm*
+   - src/udev/udev-builtin-blkid.c
+   - src/udev/udev-builtin.h
+   - src/udev/udev-builtin-input_id.c
+   - src/udev/udev-builtin-kmod.c
+   - src/udev/udev-builtin-path_id.c
+   - src/udev/udev-builtin-uaccess.c
+   - src/udev/udev-builtin-usb_id.c
+   - src/udev/udev-ctrl.h
+   - src/udev/udevd.c
+   - src/udev/udevd.h
+   - src/udev/udev-event.c
+   - src/udev/udev-event.h
+   - src/udev/udev-node.c
+   - src/udev/udev-node.h
+   - src/udev/udev-rules.c
+   - src/udev/udev-rules.h
+   - src/udev/udev-watch.c
+   - src/udev/udev-watch.h
+   - src/udev/v4l_id/*
```

Comment:
I think this is bound to get out of date quickly… E.g. Yu has a PR open to rename some files there (not those particular ones, but we do renames quite often…).  Maybe just say "some udev sources under src/udev/ are licensed under **GPL-2.0-or-later**".

Reply:
ok, I'll also specifically mentions that this means udev binaries are GPL2+ (without L)